### ISSUE TITLE
Combine executables into one folder?

### ISSUE THREAD

**Issue by iffy-pi** [iffy-pi/pushbullet-pc-integration#9]
Instead of having separate folders for each executable, we should try and combine them into one folder to reduce the amount of duplicate files we have.

- Concern: How will our shared and temp folder work in all of this? Test it

**Comment by iffy-pi** [comment]
Turns out the steps to do this is pretty easy. We simply just use a Multiple COLLECT statement (see https://www.zacoding.com/en/post/pyinstaller-create-multiple-executables/ ). That is, rather than having individual spec files, we combine them into one singular spec file but collect multiple executables.

I made the change and verified it did not break any current functionality of the application so I proceeded with more changes:
- Now all executables output to single folder WinPushBullet, instead of individual folders such as PB, PC_PullBullet etc.
- I also flattened out the installation directory, now the selected installation folder will contain the executables and the DLLs, rather than being the parent folder to the individual folders
- Following these changes I updated the hotkey template and the registry modifications in the NSIS. As a bit of cleanup I've also reconfigured the script to delete the hotkey template after it is installed.
